pennylane/measurements/measurements.py

```diff
@@ -525,7 +528,9 @@ def wires(self):
         if self.obs is not None:
             return self.obs.wires
 
-        return Wires.all_wires(self._wires) if isinstance(self._wires, list) else self._wires
+        wires = self._wires or Wires([])
```

Comment:
Calling `MeasurementProcess.wires` still returns `Wires([])` if self._wires is `None`.